pkg/rule/repo.go

```diff
@@ -171,20 +180,38 @@ func (r Rule) Run(input Parameters, ctx Context, decisions []int) (Parameters, [
 		}
 	}
 
-	params := Parameters{}
+	var (
+		params = Parameters{}
+		d      = decisions
```

Comment:
Instead of assigning the passed in `decisions` we could just construct an empty `[]int` otherwise we going to return more than one decisionwith the append happening on line 202.